exchange/exchangetest/request-multi-bidders-one-no-resp-test1.json

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
Small change, but I think the filename should be changed to mark that this test is checking for debug info, just to make the purpose of this test file clearer.

Reply:
Done!